numpy/core/src/umath/fast_loop_macros.h

```diff
@@ -86,71 +88,74 @@
 
 /*
  * loop with contiguous specialization
- * op should be the code storing the result in `tout * out`
+ * val should be the value to be stored in `tout *out`
  * combine with NPY_GCC_OPT_3 to allow autovectorization
  * should only be used where its worthwhile to avoid code bloat
  */
-#define BASE_OUTPUT_LOOP(tout, op) \
+#define BASE_OUTPUT_LOOP(tout, val) \
```

Comment:
While we're touching this - what are your thoughts on renaming it to `UNARY_CONSTANT_LOOP`? This is describing `(arg) -> const`, I'd expect `OUTPUT_LOOP` to describe `() -> result`, possibly used in something like `random` if that ever becomes a ufunc.

Reply:
Works for me - are you thinking `s/OUTPUT_/UNARY_CONSTANT_/` for everything in these two files?